inst/tests/tests.Rraw

```diff
@@ -8394,7 +8402,11 @@ x = data.table(-0.000189921844659375) # tolerance in action
 y = data.table(-0.000189921844655161)
 test(1613.561, all(all.equal(x, y, ignore.row.order = FALSE), all.equal(x, y, ignore.row.order = TRUE)))
 test(1613.562, all(is.character(all.equal(x, y, ignore.row.order = FALSE, tolerance = 0)), is.character(all.equal(x, y, ignore.row.order = TRUE, tolerance = 0))))
-test(1613.563, all(all.equal(rbind(x,y), rbind(y,y), ignore.row.order = FALSE), all.equal(rbind(x,y), rbind(y,y), ignore.row.order = TRUE)))
+test(1613.563, all(
+  all.equal(rbind(x,y), rbind(y,y), ignore.row.order=FALSE),
+  all.equal(rbind(x,y), rbind(y,y), ignore.row.order=TRUE),
+  all.equal(rbind(y,y), rbind(x,y), ignore.row.order=TRUE)
```

Comment:
New test switches the order to hit the `first-but-not-second-has-dups` branch whose `second-but-not-first` counterpart is hit by the 2nd test already